tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
```

Comment:
This is trimmed down from the defaults provided by `tsc --init`, and was included for the benefit of the test-runner.